stdlib/public/SwiftShims/RefCount.h

```diff
@@ -432,7 +431,14 @@ class RefCountBitsT {
   }
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  RefCountBitsT(RefCountBitsT<RefCountIsInline> newbits) {
+  RefCountBitsT(int dummy, RefCountBitsT<RefCountIsInline> newbits) {
```

Comment:
We could write the inline to out-of-line conversion as a static function and remove this initializer / copy constructor. It might be used infrequently enough to be reasonable.